Core/Src/epaperTask.c

```diff
@@ -1,19 +1,58 @@
+/* ==================================================================== */
+/* ============================= INCLUDES ============================= */
+/* ==================================================================== */
+
+#include <stdbool.h>
 #include "main.h"
+#include "bms.h"
 #include "epaperTask.h"
-#include "EPD_Test.h"
-#include <stdbool.h>
+#include "epaper.h"
+
+/* ==================================================================== */
+/* ========================= LOCAL VARIABLES ========================== */
+/* ==================================================================== */
 
 bool init = true;
+uint32_t lastDataUpdate = 0;
+uint32_t numUpdates = 0;
+
+/* ==================================================================== */
+/* ======================= EXTERNAL VARIABLES ========================= */
+/* ==================================================================== */
+
+extern Bms_S gBms;
+
+/* ==================================================================== */
+/* =================== GLOBAL FUNCTION DEFINITIONS ==================== */
+/* ==================================================================== */
 
 void runEpaper()
 {
     if(init)
     {
-        epdClear();
+        // Initialize the epaper display and perform a full refresh
+        epdInit();
```

Comment:
Can this be split up into an `initEpaper()` and `runEpaper()`? Then we can call the init function inside of the main.c task `StartEPaper()` 
We should probably do this for all tasks